R/commit.R

```diff
@@ -1,22 +1,26 @@
 #' Stage and commit changes
 #'
-#' @description To commit changes, start with *staging* the files to be included
-#' in the commit using [git_add()] or [git_rm()]. Use [git_status()] to see an
-#' overview of staged and unstaged changes, and finally [git_commit()] creates
-#' a new commit with currently staged files.
+#' @description
+#' To commit changes, start by *staging* the files to be included in the commit
+#' using `git_add()` or `git_rm()`. Use `git_status()` to see an overview of
```

Comment:
Here and elsewhere I did the usual un-linking of function names that are actually documented in this topic, i.e. replaced square brackets with backticks.